python/adjoint/filters.py

```diff
@@ -3,14 +3,15 @@
 """
 
 import numpy as np
-from autograd import numpy as npa
+import jax
+from jax import numpy as npj
```

Comment:
Just FYI- convention is `import jax.numpy as jnp` rather than `npj`: https://jax.readthedocs.io/en/latest/notebooks/quickstart.html